test/lib/yarn.test.ts

```diff
@@ -205,29 +201,24 @@ for (const version of ['yarn1', 'yarn2']) {
       );
       t.fail('Expected TreeSizeLimitError to be thrown');
     } catch (err) {
-      if (version === 'yarn2') {
-        t.equals(err.constructor.name, 'UnsupportedError');
-        t.equals(err.message, yarn2Error);
-      } else {
-        t.equals(err.constructor.name, 'TreeSizeLimitError');
-      }
+      t.equals(err.constructor.name, 'TreeSizeLimitError');
     } finally {
       config.YARN_TREE_SIZE_LIMIT = 6.0e6;
     }
   });
 }
 
 // // Yarn v2 specific test
-// test('.yarnrc.yaml is missing, but still resolving to yarn2 version', async (t) => {
-//   if (getRuntimeVersion() === 8) {
-//     return t.skip();
-//   }
-//
-//   const depTree = await buildDepTreeFromFiles(
-//     `${__dirname}/fixtures/missing-dot-yarnrc-yarn2/`,
-//     'package.json',
-//     `yarn.lock`,
-//   );
-//
-//   t.equal(depTree.meta?.packageManagerVersion, '2', 'resolved to yarn v2');
-// });
+test('.yarnrc.yaml is missing, but still resolving to yarn2 version', async (t) => {
+  if (getRuntimeVersion() === 8) {
```

Comment:
Not sure of the context of this test but are we still running tests in node 8/10? 
I.e. do we still need this check?

Reply:
This can be removed. We have retired node8 on CLI